packages/framework/src/Form/DisplayOnlyDomainUrlType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class DisplayOnlyDomainUrlType extends AbstractType
```

Comment:
I have a bad feeling about this new display-only-type.

I like how you made it generalized (with `route` and `route_params` and the optional `name`). But I feel like it should be a single type along with `DisplayOnlyUrlType` - there is no real need for having them separate, the `domain_id` could be optional as well.

I would mean a BC break though - is it worth it? I think that in this case yes, otherwise the `DisplayOnlyUrlType` would be only usable for routes with id as a parameter.

When we're at it, I would think about changing the `name` parameter by more appropriate `label` or `text`.

Reply:
Either way, the new form element or the changes to the existing one should be described in [the docs](https://github.com/shopsys/shopsys/blob/v7.0.0-beta2/docs/wip_glassbox/form-extension.md#what-can-we-use-for-creating-our-own-forms).

Reply:
Done